config/routes.rb

```diff
@@ -1,10 +1,19 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :genres
-  resources :movies
   resources :users
   
+  get '/movies', to: "movies#index"
+  get '/movies/:id', to: "movies#show"
+
   get '/users/:id/lists', to:'lists#index'
+  get '/lists/:id', to:'lists#show'
+  post '/lists', to:'lists#create'
+  put '/lists/:id', to:'lists#update'
+  delete '/lists/:id', to:'lists#destroy'
+  put '/movies/:movie_id/lists/:list_id', to:'lists#add_movie'
```

Comment:
I think it makes more sense to change this route to be `/lists/:list_id/movies/:movie_id` to go from the more generic info (which list your modifying) to the more specific one (which movie you're adding to that list). Same logic with route below